pkg/catalog/routes_test.go

```diff
@@ -421,6 +424,47 @@ func TestListAllowedOutboundServicesForIdentity(t *testing.T) {
 	}
 }
 
+func TestListMeshServices(t *testing.T) {
+	assert := assert.New(t)
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockKubeController := k8s.NewMockController(mockCtrl)
+	mc := MeshCatalog{
+		kubeController: mockKubeController,
+	}
+
+	testCases := []struct {
+		name     string
+		services map[string]string // name: namespace
+	}{
+		{
+			name:     "services exist in mesh",
+			services: map[string]string{"bookstore": "bookstore-ns", "bookbuyer": "bookbuyer-ns", "bookwarehouse": "bookwarehouse"},
```

Comment:
Could you directly use a list of MeshServices here?

Reply:
Also, this is more work, but it would be nice if the Mesh Services in pkg/test used individual namespaces so that we could just reuse those objects here and elsewhere